mantidproject/mantid
# Mantid Basic Course Updates

**Description of work.**
Updated images related to Plots and Settings
Improved instructions and explanations in specific parts of the Basic Course 
Also aided understanding with a change to ConvertUnits docs

**To test:**

Build docs-html
Check that the modified pages look sensible and the new explanations / images are appropriate!
See #29455 for more details.

Fixes #29455 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
